.github/workflows/release.yml

```diff
@@ -0,0 +1,263 @@
+name: Release
+
+on:
+  # push:
+  #   branches: [auto-release-workflows]
+  workflow_dispatch:
+
+jobs:
+    
+  release:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      RSPM: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"
+
+    steps:
+      - name: Setup Git repository
+        uses: actions/checkout@v4
+
+      - name: Update Ubuntu packages
+        run: sudo apt-get update
+        
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 'release'
+
+      - name: Setup pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+        with:
+            pandoc-version: '2.14.0.3'
+
+      - name: Install rmarkdown
+        run: install.packages("rmarkdown")
+        shell: Rscript {0}
+
+# Get previous version and then create the next version   
+      - name: Application Version
+        id: version
+        uses: paulhatch/semantic-version@v5.0.2
+        with:
+          tag_prefix: "v"
+          version_format: "v${major}.${minor}.${patch}"
+
+# Create version.txt to use when changing the version in the index.md
+      - name: Create version.txt
+        run: echo ${{ steps.version.outputs.version }} > version.txt
+
+# Update version and date in SS330_User_Manual.tex
+      - name: Update version and date in manual
+        run: |
+          version <- readLines("version.txt", encoding = "UTF-8")
+          version <- gsub("v", "", version)
+          manual_tex <- readLines("SS330_User_Manual.tex")
+          manual_tex[grep("Version", manual_tex)] <- gsub("[0-9].[0-9][0-9].[0-9][0-9]", version, manual_tex[grep("Version", manual_tex)])
+          date_line <- manual_tex[grep("date[{]", manual_tex)]
+          todays_month <- format(Sys.Date(), "%B")
+          todays_day <- format(Sys.Date(), "%d")
+          todays_year <- format(Sys.Date(), "%Y")
+          todays_date <- paste0("{", todays_month, " ", todays_day, ", ", todays_year, "}")
+          date_line <- gsub("\\{[^{}]*\\}", todays_date, date_line)
+          manual_tex[grep("date[{]", manual_tex)]<- date_line
+          writeLines(manual_tex, "SS330_User_Manual.tex")
+        shell: Rscript {0}
+      
+      - name: Commit date and version changes in SS330_User_Manual.tex
+        uses: EndBug/add-and-commit@v9
+        with: 
+          add: './docs/SS330_User_Manual.tex'
+          message: 'update version and date in manual'
```

Comment:
maybe change the message to `'release: update version and date in manual'. If we use `'release: ...'` in all commits related to a release then it will be easier to search the git history for them. But this is not necessary, just a personal preference.